### PR TITLE
Fix `isFullyEvolved` for Eevee evolutions.

### PR DESCRIPTION
**Pull request by gshakhn**
**Changes made:**

Fix `isFullyEvolved` for Eevee evolutions. 

This method was returning false for some evolutions since Eevee can evolve into 3 things.

### CODE REVIEW

File: library/src/main/java/com/pokegoapi/api/pokemon/EvolutionInfo.java [hunk 1]
```diff
@@ -242,7 +243,6 @@
 	private static final PokemonId[] DRATINI_EVOLUTION = {DRATINI, DRAGONAIR, DRAGONITE};
 	private static final PokemonId[] MEWTWO_EVOLUTION = {MEWTWO};
 	private static final PokemonId[] MEW_EVOLUTION = {MEW};
-
```
Comment (FabianTerhorst) [inline library/src/main/java/com/pokegoapi/api/pokemon/EvolutionInfo.java:245]: why you removed the space here?

**Comment by FabianTerhorst** [comment]
Readd the removed space and it can be merged.

**Comment by FabianTerhorst** [comment]
I pulled your commit and fixed it. #637

**Comment by gshakhn** [comment]
Didn't even notice the space difference. Sorry about that.

Thanks for fixing it!

**Pull request closed without merging**